pages/post/[slug].js

```diff
@@ -50,62 +76,94 @@ export default function Post() {
           });
         }}
       />
-      <section className="py-5">
+
+      <section className="container my-16 font-product-sans">
         <div>
           {post == null ? (
             ""
           ) : status == config.NOT_FOUND ? (
             <div className="text-xl text-center">There is no any posts.</div>
           ) : (
-            <div
-              key={post.id}
-              className="container flex  flex-col  sm:px-[4rem] md:px-[8rem] lg:px-[10rem] xl:px-[12rem] 2xl:px-[17rem]"
-            >
-              <div className="pt-14 lg:pt-0">
-                <div className="flex flex-col md:flex-row pb-12 cursor-pointer">
-                  <div className="relative w-[60px] h-[60px] md:w-[80px] md:h-[80px] lg:w-[80px] lg:h-[80px] mb-2 md:mb-0">
-                    <Image
-                      className="rounded-full h-full w-full object-cover absolute inset-0"
-                      width={200}
-                      height={200}
-                      src={authorImage}
-                      alt={authorAltText}
-                    />
-                  </div>
-                  <div className="md:pl-4 flex flex-col justify-center">
-                    <div className="text-lg text-gray-600">
-                      {post.author_id.data.attributes.username}
-                    </div>
-                    <div className="text-gray-500">
-                      <span className="">{post.publishedAt}</span>
-                      <spn className=" after:content-['\00B7'] after:mx-1 "></spn>
-                      <span>{post.readingTime} min read</span>
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div>
-                <div className="pb-3 text-4xl font-bold text-black">
-                  {post.title}
-                </div>
-                <div className="relative overflow-hidden transition-all bg-gray-100 rounded-md dark:bg-gray-800 aspect-video">
+            <div key={post.id} className="flex flex-col space-y-20 ">
+              <div className="grid grid-flow-row xl:grid-flow-col gap-10 xl:gap-8 w-90 h-90 rounded-3xl md:bg-[#14161E] md:py-20 md:px-10 xl:py-14 xl:px-8 ">
+                <div className="md:container w-full h-auto sm:h-[18rem] md:h-[22rem] lg:h-[30rem] xl:w-[35rem] xl:h-[20rem] 2xl:w-[42rem] 2xl:h-[23rem] ">
```

Comment:
can you keep all heights together in sequence like h-auto,sm:h-[18rem],md:h-[22rem]... for better readability.